src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -16,6 +16,9 @@ import UserList from "components/molecules/UserList";
 
 import { RoomModalOngoingEvent, ScheduleItem } from "..";
 
+import { IFRAME_ALLOW } from "settings";
+import "components/templates/PosterPage/components/IntroVideoPreviewModal/IntroVideoPreviewModal.scss";
```

Comment:
What are these styles imported here for? We don't tend to 'cross-import' component specific styles from a different/unrelated component like this.

Reply:
Though looking at the `IntroVideoPreviewModal` component, it seems practically identical to the  implementation here, except that it sets the `url` to the `introVideoUrl` prop (which is passed into it anyway).

So why wouldn't we just reused that component in its entirety here?

In doing so, it would pretty much make `RoomModalVideo` an irrelevant component.

Reply:
now implemented in refactored version